pyproject.toml

```diff
@@ -57,6 +57,71 @@ libcst = "^0.4.9"
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
 
+[tool.mypy]
+python_version = "3.7"
+
+# Increase our expectations
+
+check_untyped_defs          = true
+disallow_incomplete_defs    = true
+disallow_untyped_defs       = true
+no_implicit_optional        = true
+show_column_numbers         = true
+show_error_codes            = true
+strict_optional             = true
+warn_no_return              = true
+warn_redundant_casts        = true
+warn_return_any             = true
+warn_unreachable            = true
+warn_unused_ignores         = true
+
+# These are too strict for us at the moment
+
+disallow_any_decorated      = true
+disallow_any_explicit       = true
+disallow_any_expr           = true
+disallow_any_generics       = true
+disallow_any_unimported     = false
+disallow_subclassing_any    = false
+disallow_untyped_calls      = true
+disallow_untyped_decorators = true
+strict_equality             = true
```

Comment:
Is that comment accurate? Looks like most of them are enabled.

Reply:
I copied that from mypy.ini but it looks out of date.